sui_programmability/examples/games/tests/randomness_based_lottery_tests.move

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[test_only]
+module games::randomness_based_lottery_tests {
+    use sui::test_scenario::{Self, Scenario};
+    use games::randomness_based_lottery::{Self, Lottery, Ticket};
+    use sui::sui::SUI;
+    use sui::coin;
+    use sui::transfer;
+    use sui::coin::Coin;
+    use sui::randomness::Randomness;
+    use std::option;
+    use sui::randomness;
+
+    fun mint(addr: address, amount: u64, scenario: &mut Scenario) {
+        transfer::transfer(coin::mint_for_testing<SUI>(amount, test_scenario::ctx(scenario)), addr);
+        test_scenario::next_tx(scenario, addr);
+    }
+
+    #[test]
+    fun test_play_randomness_lottery() {
+        let user1 = @0x0;
+        let user2 = @0x1;
+        let user3 = @0x2;
+        let user4 = @0x3;
```

Comment:
I often do it with a tuple but that's more of a personal style thing...
Just saying there's an option 😏 

```rust
let (user1, user2, user3, user4) = (@0x0, @0x1, @0x2, @0x3, @0x4);
```